stan/math/rev/mat/functor/adj_jac_apply.hpp

```diff
@@ -48,6 +48,106 @@ template <class F, class Tuple>
 constexpr auto apply(const F& f, const Tuple& t) {
   return apply_impl(f, t, std::make_index_sequence<std::tuple_size<Tuple>{}>{});
 }
+
+/**
+ * build_y_adj takes the adjoint from the vari pointed to
```

Comment:
You don't need to repeat the name of the function. This should just say:

`Set y_adj to the the value of the adjoint of y_vi[0]`.